src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -68,30 +92,40 @@ const RenderGroupedIcons = ({ tasks }) => {
   return (
     <>
       {otherTasks.map(({ count, umbrellaStatus }) => (
-        <GroupedTaskStatusBadgeWrapper>
-          <GroupedTaskStatusBadge
-            status={umbrellaStatus}
-            key={`${umbrellaStatus}_groupedBadge`}
-            count={count}
-          />
+        <GroupedTaskStatusBadgeWrapper data-cy="grouped-task-status-badge">
+          <Link
+            to={getVersionRoute(versionId, {
+              statuses: mapUmbrellaStatusToQueryParam[umbrellaStatus],
+              variant: applyStrictRegex(buildVariantId),
+            })}
+          >
+            <GroupedTaskStatusBadge
+              status={umbrellaStatus}
+              key={`${umbrellaStatus}_${versionId}_${buildVariantId}_groupedBadge`}
```

Comment:
Nice

Reply:
Since `versionId` and `buildVariantId` are fixed for all elements of this array and [keys must only be unique amongst siblings](https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings), we could actually omit `versionId` and `buildVariantId` since key stability will have to be guaranteed by `umbrellaStatus` either way.